### PR TITLE
[RDY] Moved bank mapgen to json

### PR DESCRIPTION
**Pull request by ZhilkinSerg**

### CODE REVIEW

**Comment by John-Candlebury** [comment]
The old spawn system allowed for more than a single item to spawn on each safe. I think it would be a good idea to let that happen in here, after all the items inside vaults are some of the rarest in game

File: src/mapgen_functions.cpp [hunk 1]
```diff
@@ -3494,182 +3494,6 @@ void mapgen_police(map *m, oter_id terrain_type, mapgendata dat, int, float dens
...
-    }
-    if (one_in(2)) {
-        line_furn(m, f_chair, 17, 2, 19, 2);
-        m->furn_set(18, 2, f_table);
-    }
-    if (one_in(2)) {
-        m->furn_set(21, 2, f_indoor_plant);
-    }
-    //Windows or glass wall front?
-    int tmp = 0;
```
Comment (BorkBorkGoesTheCode) [inline src/mapgen_functions.cpp]: Is there any way to retain the old random terrain generation?

Comment (Coolthulhu) [reply src/mapgen_functions.cpp]: Multiple definitions - one per variant.

Comment (ZhilkinSerg) [reply src/mapgen_functions.cpp]: Indoor plants are random - they can be spawned with 50% probability. I've also restored random locked doors.

Other randomness is not possible now in JSON - only in LUA.

**Comment by ZhilkinSerg** [comment]
> The old spawn system allowed for more than a single item to spawn on each safe. I think it would be a good idea to let that happen in here, after all the items inside vaults are some of the rarest in game

Wasn't old loot system broken?

**Comment by Zireael07** [comment]
> Wasn't old loot system broken?

I think it was the usage that was broken, not the old system in general.

And yes, more than 1 item in a safe, please.

**Comment by ZhilkinSerg** [comment]
Increased quantity of items in safes.

**Pull request merged**